content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -257,9 +257,7 @@ sandbox image by setting the following config:
 You might need to restart `containerd` as well once you've updated the config file: `systemctl restart containerd`.
 
 Please note, that it is a best practice for kubelet to declare the matching `pod-infra-container-image`.
-If not configured, kubelet may attempt to garbage collect the `pause` image.
-There is ongoing work in [containerd to pin the pause image](https://github.com/containerd/containerd/issues/6352)
-and not require this setting on kubelet any longer.
+If not configured, kubelet may attempt to garbage collect the `pause` image..
```

Comment:
Please also remove the unnecessary full stop that seems to have been left behind.

```suggestion
If not configured, kubelet may attempt to garbage collect the `pause` image.
```

Reply:
The issue is fixed, so this setting is not required, correct? Should we remove the whole chapter then?